src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.DateTimes.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_DateTimes
+    {
+        private const int DataSize = 100_000;
+
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+        private PooledBufferWriter<byte> _pooledBufferWriter;
+
+        private DateTime[] _dateArrayValues;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [Params(true, false)]
+        public bool Pool;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+            _pooledBufferWriter = new PooledBufferWriter<byte>();
+
+            _dateArrayValues = new DateTime[DataSize];
+
+            for (int i = 0; i < DataSize; i++)
+            {
+                _dateArrayValues[i] = DateTime.Now;
```

Comment:
Can we have a fixed `DateTime` instead of a changing one every time we run?

Reply:
Yes, will do. However, I would like to generate different DateTimes in my array. Any good way to do it?

Reply:
What's the value of different DateTimes? Does the json writer parse the date and do things differently if they vary?

If you think it is interesting, you could pick some date and fill the array with `.AddDays` or the like.